book/src/dev/rfcs/0010-v5-transaction.md

```diff
@@ -416,6 +416,8 @@ bitflags! {
 
 This type is also defined in `orchard/shielded_data.rs`.
 
+Note: A [consensus rule](https://zips.z.cash/protocol/protocol.pdf#txnencodingandconsensus) was added to the protocol specification stating that the reserved bits (2..7) of the flags must be all zero in transactions V5.
```

Comment:
Please quote the exact consensus rule from the spec.

If we copy and paste the consensus rule, we can compare the design to the new rule when there are spec changes.

```suggestion
Note: A [consensus rule](https://zips.z.cash/protocol/protocol.pdf#txnencodingandconsensus) was added to the protocol specification stating that:
> In a version 5 transaction, the reserved bits 2..7 of the flagsOrchard field MUST be zero.
```